ejercicios/ejercicio-100/ejercicio.py

```diff
@@ -9,12 +9,20 @@
 
 
 def mas_de_50(path):
-    """ 
-    Abrir el archivo con el libro de texto, contar la
-    repeticion de palabras y devolver solo aquellas que
-    cumplen con los requisitos
-    """
-    pass
+    f = open(path, encoding="utf8")
+    text = f.read()
+    f.close()
+
+    palabras = {}
+    for palabra in text.split(' '):
+        palabra = palabra.lower()
+        if len(palabra) < 6:  # omitir las palabras con menos de 6 letras
+            continue
+        if palabra not in palabras.keys():
+            palabras[palabra] = 1
+        palabras[palabra] += 1
```

Comment:
De esta forma va a inicializar en 1 y despues sumar 1 inmediatamente
O se inicializa en cero:

```suggestion
        if palabra not in palabras.keys():
            palabras[palabra] = 0
        palabras[palabra] += 1
```

... o se hace con else

```suggestion
        if palabra not in palabras.keys():
            palabras[palabra] = 1
        else:
            palabras[palabra] += 1
```